[2022-06-05 14:21:44,186] {scheduler_job.py:182} INFO - Started process (PID=340) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:21:44,187] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:21:44,188] {logging_mixin.py:104} INFO - [2022-06-05 14:21:44,188] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:21:44,208] {logging_mixin.py:104} INFO - [2022-06-05 14:21:44,206] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:21:44,211] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:21:44,223] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.039 seconds
[2022-06-05 14:22:14,466] {scheduler_job.py:182} INFO - Started process (PID=371) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:22:14,469] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:22:14,470] {logging_mixin.py:104} INFO - [2022-06-05 14:22:14,469] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:22:14,489] {logging_mixin.py:104} INFO - [2022-06-05 14:22:14,486] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:22:14,491] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:22:14,505] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.045 seconds
[2022-06-05 14:22:44,592] {scheduler_job.py:182} INFO - Started process (PID=402) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:22:44,593] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:22:44,593] {logging_mixin.py:104} INFO - [2022-06-05 14:22:44,593] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:22:44,611] {logging_mixin.py:104} INFO - [2022-06-05 14:22:44,609] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:22:44,614] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:22:44,629] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.040 seconds
[2022-06-05 14:23:14,758] {scheduler_job.py:182} INFO - Started process (PID=433) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:23:14,759] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:23:14,760] {logging_mixin.py:104} INFO - [2022-06-05 14:23:14,760] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:23:14,777] {logging_mixin.py:104} INFO - [2022-06-05 14:23:14,775] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:23:14,780] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:23:14,792] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.037 seconds
[2022-06-05 14:23:45,011] {scheduler_job.py:182} INFO - Started process (PID=514) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:23:45,012] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:23:45,013] {logging_mixin.py:104} INFO - [2022-06-05 14:23:45,013] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:23:45,030] {logging_mixin.py:104} INFO - [2022-06-05 14:23:45,028] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:23:45,034] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:23:45,049] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.040 seconds
[2022-06-05 14:24:15,205] {scheduler_job.py:182} INFO - Started process (PID=558) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:24:15,206] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:24:15,207] {logging_mixin.py:104} INFO - [2022-06-05 14:24:15,207] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:24:15,222] {logging_mixin.py:104} INFO - [2022-06-05 14:24:15,219] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:24:15,224] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:24:15,237] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.034 seconds
[2022-06-05 14:24:45,351] {scheduler_job.py:182} INFO - Started process (PID=589) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:24:45,352] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:24:45,352] {logging_mixin.py:104} INFO - [2022-06-05 14:24:45,352] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:24:45,368] {logging_mixin.py:104} INFO - [2022-06-05 14:24:45,365] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:24:45,371] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:24:45,385] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.037 seconds
[2022-06-05 14:25:15,934] {scheduler_job.py:182} INFO - Started process (PID=620) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:25:15,935] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:25:15,936] {logging_mixin.py:104} INFO - [2022-06-05 14:25:15,936] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:25:15,951] {logging_mixin.py:104} INFO - [2022-06-05 14:25:15,949] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:25:15,954] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:25:15,968] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.036 seconds
[2022-06-05 14:25:46,044] {scheduler_job.py:182} INFO - Started process (PID=651) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:25:46,045] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:25:46,046] {logging_mixin.py:104} INFO - [2022-06-05 14:25:46,046] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:25:46,063] {logging_mixin.py:104} INFO - [2022-06-05 14:25:46,060] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:25:46,066] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:25:46,081] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.039 seconds
[2022-06-05 14:26:16,138] {scheduler_job.py:182} INFO - Started process (PID=682) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:26:16,139] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:26:16,140] {logging_mixin.py:104} INFO - [2022-06-05 14:26:16,140] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:26:16,164] {logging_mixin.py:104} INFO - [2022-06-05 14:26:16,160] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:26:16,168] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:26:16,187] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.053 seconds
[2022-06-05 14:26:46,334] {scheduler_job.py:182} INFO - Started process (PID=713) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:26:46,335] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:26:46,336] {logging_mixin.py:104} INFO - [2022-06-05 14:26:46,336] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:26:46,354] {logging_mixin.py:104} INFO - [2022-06-05 14:26:46,351] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:26:46,356] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:26:46,370] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.037 seconds
[2022-06-05 14:27:16,522] {scheduler_job.py:182} INFO - Started process (PID=744) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:27:16,523] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:27:16,524] {logging_mixin.py:104} INFO - [2022-06-05 14:27:16,524] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:27:16,541] {logging_mixin.py:104} INFO - [2022-06-05 14:27:16,538] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:27:16,543] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:27:16,555] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.035 seconds
[2022-06-05 14:27:46,591] {scheduler_job.py:182} INFO - Started process (PID=775) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:27:46,592] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:27:46,592] {logging_mixin.py:104} INFO - [2022-06-05 14:27:46,592] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:27:46,609] {logging_mixin.py:104} INFO - [2022-06-05 14:27:46,606] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:27:46,612] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:27:46,624] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.035 seconds
[2022-06-05 14:28:16,730] {scheduler_job.py:182} INFO - Started process (PID=806) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:28:16,731] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:28:16,731] {logging_mixin.py:104} INFO - [2022-06-05 14:28:16,731] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:28:16,746] {logging_mixin.py:104} INFO - [2022-06-05 14:28:16,744] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:28:16,749] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:28:16,763] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.036 seconds
[2022-06-05 14:28:46,931] {scheduler_job.py:182} INFO - Started process (PID=837) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:28:46,932] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:28:46,933] {logging_mixin.py:104} INFO - [2022-06-05 14:28:46,933] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:28:46,948] {logging_mixin.py:104} INFO - [2022-06-05 14:28:46,946] {dagbag.py:305} ERROR - Failed to import: /opt/airflow/dags/example_twitter_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 302, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/example_twitter_dag.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/hive_operator.py", line 23, in <module>
    from airflow.providers.apache.hive.operators.hive import HiveOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-06-05 14:28:46,950] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:28:46,964] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.036 seconds
[2022-06-05 14:29:17,522] {scheduler_job.py:182} INFO - Started process (PID=868) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:29:17,523] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:29:17,524] {logging_mixin.py:104} INFO - [2022-06-05 14:29:17,524] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:29:17,525] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:29:17,525] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.005 seconds
[2022-06-05 14:29:48,094] {scheduler_job.py:182} INFO - Started process (PID=899) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:29:48,095] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:29:48,096] {logging_mixin.py:104} INFO - [2022-06-05 14:29:48,096] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:29:48,097] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:29:48,097] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.005 seconds
[2022-06-05 14:30:18,151] {scheduler_job.py:182} INFO - Started process (PID=930) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:30:18,152] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:30:18,152] {logging_mixin.py:104} INFO - [2022-06-05 14:30:18,152] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:30:18,153] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:30:18,153] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.005 seconds
[2022-06-05 14:30:48,692] {scheduler_job.py:182} INFO - Started process (PID=961) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:30:48,693] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:30:48,693] {logging_mixin.py:104} INFO - [2022-06-05 14:30:48,693] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:30:48,694] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:30:48,695] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.006 seconds
[2022-06-05 14:31:18,715] {scheduler_job.py:182} INFO - Started process (PID=1088) to work on /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:31:18,717] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/example_twitter_dag.py for tasks to queue
[2022-06-05 14:31:18,718] {logging_mixin.py:104} INFO - [2022-06-05 14:31:18,717] {dagbag.py:448} INFO - Filling up the DagBag from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:31:18,718] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/example_twitter_dag.py
[2022-06-05 14:31:18,719] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/example_twitter_dag.py took 0.006 seconds
